app/models/membership.rb

```diff
@@ -2,4 +2,15 @@ class Membership < ApplicationRecord
   belongs_to :user
   belongs_to :group
   belongs_to :creator, class_name: "User"
+
+  validate :user_and_group_in_same_organisation
```

Comment:
Can we update the commit message for 80f4cd86a38b0df95b6009a36a04bf8a93dca8bc? It says 'Prevent a user being a member of a group more than once.', but the changes appear to be about a different validation.